tests/integration/test_charm.py

```diff
@@ -110,6 +153,40 @@ async def test_grafana_dashboards_devices(ops_test: OpsTest, mocker):
     await assert_grafana_dashboards(app, dashboards)
 
 
+async def test_loki_alert_rules_devices(ops_test: OpsTest, mocker):
+    """Test Loki alert rules for devices are defined in relation data bag."""
+    async with ops_test.fast_forward():
+        await ops_test.model.wait_for_idle(apps=[APP_NAME], status="active", timeout=120)
```

Comment:
Why this 'wait' again ?

Reply:
We wait for it to reach active again after the fast_forward and the update hook.
https://ops.readthedocs.io/en/latest/howto/write-integration-tests-for-a-charm.html#speed-up-update-status-with-fast-forward